tests/avantis-realtime-salary.js

```diff
@@ -210,4 +212,59 @@ describe('avantis-realtime-salary', () => {
   });
 
 
+  it('Employee can claim all of remaining salary', async () => {
+    const [salaryVaultAccountPDA, salaryVaultAccountPDABump] = await PublicKey.findProgramAddress(
+      [Buffer.from(anchor.utils.bytes.utf8.encode("salary_vault_account"))],
+      program.programId
+    );
+
+    const [salaryVaultAuthorityPDA, salaryVaultAuthorityPDABump] = await PublicKey.findProgramAddress(
+      [Buffer.from(anchor.utils.bytes.utf8.encode("salary_vault_authority"))],
+      program.programId
+    );
+
+    const [salaryStatePDA, salaryStatePDABump] = await PublicKey.findProgramAddress(
+      [employee1Keypair.publicKey.toBuffer()],
+      program.programId
+    );
+
+    let employeeAmountBeforeClaim = await mintAccount.getAccountInfo(employee1TokenAccount);
+    employeeAmountBeforeClaim = employeeAmountBeforeClaim.amount.toNumber();
+
+    //sleep for 5 seconds to wait for claimable amount
+    await sleep(5000);
+
+    const tx = await program.rpc.claimSalary(
+      {
+        accounts: {
+          claimer: employee1Keypair.publicKey,
+          employeeTokenAccount: employee1TokenAccount,
+          vaultAccount: salaryVaultAccountPDA,
+          employeeSalaryState: salaryStatePDA,
+          vaultAuthority: salaryVaultAuthorityPDA,
+          tokenProgram: TOKEN_PROGRAM_ID
+        },
+        signers: [employee1Keypair],
+      }
+    );
+
+    let employeeAmountAfterClaim = await mintAccount.getAccountInfo(employee1TokenAccount);
+    employeeAmountAfterClaim = employeeAmountAfterClaim.amount.toNumber();
+
+    let totalClaimedAmount = employeeAmountAfterClaim - employeeAmountBeforeClaim;
+
+    console.log("total claimed amount", totalClaimedAmount);
+
+    // we expect >5 token because total daily rate is 24 * 60 * 60 = 1 token per seconds
+    // we sleep 5 seconds, so total claimed should > 5
+    assert.ok(totalClaimedAmount > 5);
```

Comment:
we may add upper bound condition to make sure that totalClaimAmount not go over the estimated range, it may be like `assert.ok((totalClaimedAmount > 5) && (totalClaimedAmount < 7))`